ferrocene/doc/user-manual/src/targets/index.rst

```diff
@@ -26,14 +26,18 @@ There are also two variants of the standard library available:
 * **Full**: provides the ``core``, ``alloc``, ``std``, ``test`` and
   ``proc_macro`` built-in crates.
 
-Supported targets
+Qualified targets
 -----------------
 
-Supported targets are provided with the highest level of assurance. They are
+Qualified targets are provided with the highest level of assurance. They are
 qualified for use in safety-critical contexts (when used with a qualified
 rustc), and the full Ferrocene test suite is executed on the target for every
 code change.
 
+Only stable releases of qualified targets are qualified. Other releases, such
+as beta, should be considered Quality Managed. Specific releases can be
```

Comment:
to make it clear we are talking about the same ones mentioned in the previous sentence
```suggestion
as beta, should be considered Quality Managed. Such releases can be
```